sop-website/docs/Data-Requests/Data-Requests.mdx

```diff
@@ -3,3 +3,75 @@ title: Data Requests
 id: Data Requests
 description:  An SOP for elements related to data requests
 ---
+
+# Awaiting Approval
+
+# Data Requests
+
+# Purpose
+The purpose of this correspondence is to specify the content, format, and timeline of the data CHoRUS sites should be contributing to this first upload to the CHoRUS data repository.
+
+# Procedures
+**Number of patients**: The count is based on hospital admissions, not on patients. A given patient may therefore contribute to several encounters.  We are requesting the preparation and upload of data from 25% of your total expected contribution to the CHoRUS database.
+
+**Encounter selection**: For this first upload, we are requesting that encounters be sampled randomly from all ICU admissions since January 2019.
+
+**Data to be uploaded**: The data to be uploaded will include structured and unstructured (notes and reports) EHR elements, images and waveforms for the entire duration of an inpatient encounter (hospitalization). The hospitalization, inpatient encounter, or visit occurrence must include one or more admissions to an intensive care unit. OHDSI recommends that the admission correspond to the initial contact with a healthcare system, even if this contact was initiated in a different hospital.
+
+The data elements to be included will be the Tier 1 data elements identified by CHoRUS investigators as of primary significance across all four domains of data [Broad DA SOP](https://chorus-ai.github.io/data_acq_SOP/docs/Data-Uploading/Broad%20SOP/).
+
+1. Structured EHR Data
+    >Structured EHR data should populate all OMOP Clinical Data Tables, except the NOTE, NOTE_NLP, and FACT_RELATIONSHIP  tables. All uploads are expected to be in text format, preferably .csv. Since some text 
+    fields contain commas, we suggest enclosing all text fields within quotes “”, in accordance with .csv standards. The use of a different delimiter, such as bar “|”, is also acceptable. We require sites to run 
+    OHDSI’s Data Quality Dashboard (DQD) and ARES tools locally prior to uploading , to offer a first layer of quality control of site data. Reports from those runs should be uploaded in the main upload folder 
+    (see “Uploading the data” below).
+
+2. Unstructured EHR Data
+    >The ideal upload would be in the form of an OMOP NOTE and NOTE_NLP tables, understanding that we are not requesting actual notes but rather the standard concepts extracted from those notes using the OHNLP 
+    pipeline. Thus, the uploaded version of the NOTE table should have the note_text field left empty or NULL.  We do urge sites to produce and keep a local version of the NLP NOTE table that includes a filled 
+    note_text field.
+
+   >Below are 2 links to assistance with this process:
+    [Google Drive](https://drive.google.com/drive/folders/1bDBSF2pkx50kTyTXmWJYt1Ta6pgwh4HH) is where the recorded lectures and task 1 materials are located.
```

Comment:
Perhaps precede the two items with a `-` so they are rendered as an unordered list. If you tried this and they weren't being rendered as a list, the fix is to either (1) add an empty line after "process:" or (2) add a couple of blank spaces at the end of the line items.